docs/reference/commandline/stack_ps.md

```diff
@@ -49,9 +71,196 @@ Multiple filter flags are combined as an `OR` filter. For example,
+3s052erw88a4w5   Running             Running 31 minutes ago
+kqgdmededccb        qyprtqw1g5nrki557i974ou1d.2   dockersamples/examplevotingapp_vote:before     qaqt4nrzo77
+5jrx6detglho01   Running             Running 31 minutes ago
+t72q3z038jeh        tg61x8myx563ueo3urmn1ic6m.2   redis:alpine                                   kanqcxfajd1
+r16wlnqcblobmm   Running             Running 31 minutes ago
+```
+
+### Do not truncate output
+
+When deploying a service, docker resolves the digest for the service's
+image, and pins the service to that digest. The digest is not shown by
+default, but is printed if `--no-trunc` is used. The `--no-trunc` option
+also shows the non-truncated task ID, and error-messages, as can be seen below:
+
+```bash
+$ docker stack ps --no-trunc voting
+ID                          NAME                  IMAGE
+                                                             NODE                DESIRED STATE       CURREN
+ STATE            ERROR               PORTS
+xim5bcqtgk1bxqz91jzo4a1s5   voting_worker.1       dockersamples/examplevotingapp_worker:latest@sha256:3e4dd
+f59c15f432280a2c0679c4fc5a2ee5a797023c8ef0d3baf7b1385e9fed   node2               Running             Runnin
+ 32 minutes ago
+q7yik0ks1in6kv32gg6y6yjf7   voting_result.1       dockersamples/examplevotingapp_result:before@sha256:83b56
+996e930c292a6ae5187fda84dd6568a19d97cdb933720be15c757b7463   node1               Running             Runnin
+ 32 minutes ago
+rx5yo0866nfxc58zf4irsss6n   voting_vote.1         dockersamples/examplevotingapp_vote:before@sha256:8e64b18
+2c87de902f2b72321c89b4af4e2b942d76d0b772532ff27ec4c6ebf6     node3               Running             Runnin
+ 32 minutes ago
+tz6j82jnwrx7n2offljp3mn03   voting_db.1           postgres:9.4@sha256:6046af499eae34d2074c0b53f9a8b404716d4
+15e4a03e68bc1d2f8064f2b027                                   node1               Running             Runnin
+ 32 minutes ago
+w48spazhbmxcmbjfi54gs7x90   voting_redis.1        redis:alpine@sha256:9cd405cd1ec1410eaab064a1383d0d8854d1e
+f74a54e1e4a92fb4ec7bdc3ee7                                   node2               Running             Runnin
+ 32 minutes ago
+6jj1m02freg1n3z9n1evrzsbl   voting_visualizer.1   dockersamples/visualizer:stable@sha256:f924ad66c8e94b10ba
+af7bdb9cd491ef4e982a1d048a56a17e02bf5945401e5                node1               Running             Runnin
+ 32 minutes ago
+kqgdmededccbhz2wuc0e9hx7g   voting_vote.2         dockersamples/examplevotingapp_vote:before@sha256:8e64b18
+2c87de902f2b72321c89b4af4e2b942d76d0b772532ff27ec4c6ebf6     node2               Running             Runnin
+ 32 minutes ago
+t72q3z038jehe1wbh9gdum076   voting_redis.2        redis:alpine@sha256:9cd405cd1ec1410eaab064a1383d0d8854d1e
+f74a54e1e4a92fb4ec7bdc3ee7                                   node3               Running             Runnin
+ 32 minutes ago
+```
+
+### Only display task IDs
+
+The `-q `, or `--quiet` option only shows IDs of the tasks in the stack.
```

Comment:
Perhaps we should add an example / description why this option is useful (it's usually used for automation, such as `docker inspect $(docker stack ps -q voting)`).

I'm ok with doing that in a follow up, because I expect other documentation pages also don't mention this.